### PR TITLE
safe_name and headquarter imports

### PR DESCRIPTION
**Pull request by xithan**
* don't call OC API for headquarter imports
* use `refresh_machine_output` instead of `reset_machine_output` for deployment
* escape card name in html formats
* API to query OC industry codes

### CODE REVIEW

File: lib/capistrano/maintenance/maintenance.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    img {
+      width: 80px;
+    }
+    h1 {
+      font-weight: 300;
+    }
+  </style>
+</head>
+<body>
+<img src="http://dev.wikirate.org/animated-logo.gif"/>
```
Comment (ethn) [inline lib/capistrano/maintenance/maintenance.html.erb:21]: indentation?

Comment (ethn) [reply lib/capistrano/maintenance/maintenance.html.erb:21]: seems like we might want this image on a cloud server somewhere.

File: lib/capistrano/tasks/deploy.cap [hunk 2]
```diff
@@ -57,11 +55,21 @@ namespace :deploy do
...
       within release_path do
         execute :rake, "decko:reset_cache"
-        execute :rake, "card:reset_machine_output"
       end
     end
   end
 
+  desc "regenerate all stuff create by machine cards (like css and javascript)"
+  after :clear_cache, :reset_machine_output do
+    invoke "maintenance:enable"
```
Comment (ethn) [inline lib/capistrano/tasks/deploy.cap]: I'd very much prefer not to have to enable maintenance for routine deployments.

File: mod/csv_import/lib/import_manager/script_import_manager.rb [hunk 3]
```diff
@@ -40,4 +40,9 @@ def row_failed _csv_row
       nil
     end
   end
+
+  def log_status
+    puts "#{@current_row.row_index}:  #{@current_row.name}"
```
Comment (ethn) [inline mod/csv_import/lib/import_manager/script_import_manager.rb:45]: puts?  do we want to use Rails.logger?

File: mod/widgets/lib/open_corporates/api.rb [hunk 4]
```diff
@@ -29,13 +29,51 @@ def fetch_jurisdictions
...
+      #              "uid":"isic_4-A"}},
+      #          {"industry_code":{
+      #              "code":"01","description":"Crop and animal production",
+      #          ....
+      # This method returns an array with a hash for each industry codes
+      # @return [Array<Hash>] each industry code has a code, a description and a uid key
+      #   Example:
+      #       :code=>"855",
+      #       :description=>"Educational support activities",
+      #       :uid=>"isic_4-855"
```
Comment (ethn) [inline mod/widgets/lib/open_corporates/api.rb:49]: nice docs.

**Review by ethn — APPROVED** [review]

File: mod/csv_import/lib/import_manager/script_import_manager.rb (hunk 3 above)
Comment (xithan) [reply mod/csv_import/lib/import_manager/script_import_manager.rb:45]: no, it's for scripts

**Pull request merged**